CHANGELOG.md

```diff
@@ -101,6 +101,8 @@ Main (unreleased)
 
 - Improved resilience of graph evaluation in presence of slow components. (@thampiotr)
 
+- Remove need for windows_exporter fork. (@mattdurham)
```

Comment:
This isn't really a user-facing change, is it? Or if it is, can we rephrase this in a way which tells users the impact on them?

Reply:
I debated on whether to add to the changelog but likely it brings in more metrics, so I could workshop that a bit.

Reply:
Maybe could rephrase it as a "updated version of windows_exporter to X"?